esy.json

```diff
@@ -26,10 +28,55 @@
     "5. Copy ./scripts/esy-prepublish.js in to your repo and change packages= to your set of packages."
   ],
   "esy": {
-    "build": [["dune", "build", "-p", "reason,rtop"]],
+    "build": [
+      [
+        "dune",
+        "build",
+        "-p",
+        "reason,rtop"
+      ]
+    ],
     "install": [
-      ["esy-installer", "reason.install"],
-      ["esy-installer", "rtop.install"]
+      [
+        "esy-installer",
+        "reason.install"
+      ],
+      [
+        "esy-installer",
+        "rtop.install"
+      ]
     ]
+  },
+  "buildDirs": {
+    "test": {
+      "require": [
+        "reason/library",
+        "rely.lib"
+      ],
+      "flags": [
+        "-linkall",
+        "-g"
+      ]
+    },
+    "testExe": {
+      "require": [
+        "reason/test"
+      ],
+      "bin": {
+        "RunReasonTests.exe": "RunReasonTests.re"
+      }
+    },
+    "library": {},
+    "executable": {
+      "require": [
+        "reason/library"
+      ],
+      "bin": {
+        "ReasonApp.exe": "ReasonApp.re"
+      }
+    }
+  },
+  "scripts": {
+    "test": "esy x RunReasonTests.exe"
   }
```

Comment:
followed previous pesy test structure

Reply:
Great! The convention we suggest is to do exactly this - make an `esy x` test script, so basically every project can be cloned, then you run `esy` then `esy test`